stolon/values.yaml

```diff
@@ -45,6 +45,9 @@ store:
   backend: "etcdv3"
   ## store endpoints MUST be set!
   endpoints: "http://etcd-etcd-0.etcd-etcd:2379,http://etcd-etcd-1.etcd-etcd:2379,http://etcd-etcd-2.etcd-etcd:2379"
+  ## for kubernetes backend (experimental)
```

Comment:
Please remove this comments

Reply:
I added the RBAC stuff as well. I left the two open points in regard to the labels open and wait for your response first.